civis/resources/_resources.py

```diff
@@ -539,4 +538,26 @@ def generate_classes_maybe_cached(cache, api_key, api_version, resources):
             raise ValueError(msg.format(type(cache)))
         spec = JsonRef.replace_refs(raw_spec)
         classes = parse_api_spec(spec, api_version, resources)
+    classes = add_no_underscore_compatibility(classes)
+    return classes
+
+
+def add_no_underscore_compatibility(classes):
```

Comment:
I suggest that you make this function private. Also, the function both modifies its input and returns a value. I suggest that it do one or the other -- either it modifies the input `classes`, or it leaves `classes` unchanged and returns a new dictionary. Since the function's name starts with "add", I think it would make sense to modify the input.

Reply:
> I suggest that you make this function private. 

Agreed.  Some of the other private functions here are relying on the fact that the module is prefixed with an underscore (`_resources.py`) and aren't listed in the `__all__` of `__init__.py`.  But now that you mention it, it does seem to be a good idea to be explicit.

> Since the function's name starts with "add", I think it would make sense to modify the input.

Agreed. We can't avoid modifying the input the because `feature_flag` needs to be removed.

Reply:
It should be possible to avoid modifying the input, no? You could make a copy of the input dictionary and only modify the copy.

Reply:
Yeah, a copy is a good idea.